sql-plugin/src/main/spark311/scala/com/nvidia/spark/rapids/shims/ParquetLegacyNanoAsLongShims.scala

```diff
@@ -35,6 +35,7 @@ package com.nvidia.spark.rapids.shims
 import org.apache.hadoop.conf.Configuration
 
 object ParquetLegacyNanoAsLongShims {
+  def supportsLegacyParquetNanosAsLong(): Boolean = false
```

Comment:
If cudf is doing the right thing with nanos loaded as longs then IMHO this should return true